### PR TITLE
SCALRCORE-23367 > Deprecate Provider Credentials

### PR DESCRIPTION
**Pull request by lyzohub**
### Changelog
* [X] I have updated CHANGELOG.md according to [CONTRIBUTING.md](../CONTRIBUTING.md)

### Documentation
* [X] I have updated resource/data source documentation in [docs](../docs)

### State
* [ ] My changes affect the state
* [ ] I have included a state migration and a unit test for it

### CODE REVIEW

File: docs/resources/scalr_environment.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ resource "scalr_environment" "test" {
 * `name` - (Required) Name of the environment.
 * `account_id` - (Required) ID of the environment account, in the format `acc-<RANDOM STRING>`
 * `cost_estimation_enabled` - (Optional) Set (true/false) to enable/disable cost estimation for the environment. Default `true`.
-* `cloud_credentials` - (Optional) List of the environment cloud-credentials IDs, in the format `cred-<RANDOM STRING>`.
+* `cloud_credentials` - Deprecated. Use `default_provider_configurations` instead.
```
Comment (soltysss) [inline docs/resources/scalr_environment.md:26]: @lyzohub how about adding deprecated profile to resource relationship?

**Review by soltysss — APPROVED** [review]

File: docs/resources/scalr_environment.md (hunk 1 above)
Comment (lyzohub) [reply docs/resources/scalr_environment.md:26]: @soltysss Done! https://github.com/Scalr/fatmouse/pull/5516

File: CHANGELOG.md [hunk 2]
```diff
@@ -58,6 +58,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - scalr-server >= `8.31.0`
 
+### Deprecated
```
Comment (petroprotsakh) [inline CHANGELOG.md:61]: Why is it in `1.0.0-rc36` section?

Comment (lyzohub) [reply CHANGELOG.md]: @petroprotsakh typo

**Review by petroprotsakh — APPROVED** [review]

**Pull request merged**